src/components/Card/WCard.vue

```diff
@@ -112,8 +112,8 @@ function toggleHover() {
 }
 
 .card {
-  box-shadow: 7px 10px 0px rgb(0, 0, 0);
   position: relative;
+  @apply shadow-[7px_10px_0px] shadow-black-950 dark:shadow-wit-blue-500;
```

Comment:
The shadow and the text in the WCard with dark mode is not working

![image](https://github.com/user-attachments/assets/832a0809-c3e3-4d34-ac9d-5de58635084b)
![image](https://github.com/user-attachments/assets/45346f59-4227-4def-9705-3dd48f0f16ee)